docs/en/deployment/how_to_use_on_kubernetes.md

```diff
@@ -64,7 +67,7 @@ To install Helm, refer to the [Helm Installation Guide](https://helm.sh/docs/int
            memory: "<memory-request>"
    ```
 
-2. Check and update kubelet root directory
+3. Check and update kubelet root directory
```

Comment:
This paragraph symbol shouldn't be changed.

Reply:
Fixed, thanks for pointing out the mistake